packages/marketplace/src/components/pages/client-app-detail/client-app-detail.tsx

```diff
@@ -181,10 +181,19 @@ const ClientAppDetail: React.FC<ClientAppDetailProps> = () => {
           )}
         />
         <AppContent desktopIntegrationTypes={userDesktopIntegrationTypes} appDetailData={appDetailData} />
-        <FormSection className={classNames('is-clearfix', clientAppDetailStyles.footerContainer)}>
-          <Button className="is-pulled-right" onClick={onBackToAppsButtonClick(history, loginType)}>
-            Back To Apps
-          </Button>
+        <FormSection className={clientAppDetailStyles.footerContainer}>
+          <div className={clientAppDetailStyles.hiddenInDesktopScreenSize}>
+            <ContactDeveloperSection
+              contact={{
+                developer: appDetailData.developer,
```

Comment:
Please check line #160:
`const { id = '', installedOn = '' } = appDetailData`

Can we also unpack the values here using destructuring assignment syntax instead?